tests/pages/qanPage.js

```diff
@@ -125,8 +133,37 @@ module.exports = {
 
     _selectDetails(row) {
         I.click("//table/tr["+ (row + 1) + "]//td[2]");
+        this.waitForDetailsSection();
+    },
+
+    selectSectionInDetails(section) {
+        I.waitForElement(section, 30);
+        I.click(section);
+    },
+
+    waitForTableTabContentsLoaded() {
```

Comment:
The same feedback, I understand you want to increase readability, but having `waitForTableTabContentsLoaded` and `waitForExplainTabContentsLoaded` and then may be more in future, could you think of making a more generic function, wait for details and pass parameters? `waitForTabContentsLoaded` if it is not feasible, not a problem

Reply:
yes, i'll fix it as you say